README.md

```diff
@@ -159,6 +159,161 @@ python ./scripts/train_hf_ner.py ./data/annotations/train.spacy ./data/annotatio
+| --------------------------- | ---- | ----------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------------- |
+| `dataset`                   | str  | Prodigy dataset to save annotations to.                                                                                                         |                                     |
+| `file_path`                 | Path | Path to `.jsonl` data to annotate. The data should at least contain a `"text"` field.                                                           |                                     |
+| `labels`                    | str  | Comma-separated list defining the text categorization labels the model should predict.                                                          |                                     |
+| `--lang`, `-l`              | str  | Language of the input data - will be used to obtain a relevant tokenizer.                                                                       | `"en"`                              |
+| `--segment`, `-S`           | bool | Flag to set when examples should be split into sentences. By default, the full input article is shown.                                          | `False`                             |
+| `--model`, `-m`             | str  | GPT-3 model to use for initial predictions.                                                                                                     | `"text-davinci-003"`                |
+| `--prompt_path`, `-p`       | Path | Path to the `.jinja2` [prompt template](templates).                                                                                             | `./templates/textcat_prompt.jinja2` |
+| `--examples-path`, `-e`     | Path | Path to examples to help define the task. The file can be a .yml, .yaml or .json. If set to `None`, zero-shot learning is applied.              | `None`                              |
+| `--max-examples`, `-n`      | int  | Max number of examples to include in the prompt to OpenAI. If set to 0, zero-shot learning is always applied, even when examples are available. | 2                                   |
+| `--batch-size`, `-b`        | int  | Batch size of queries to send to the OpenAI API.                                                                                                | 10                                  |
+| `--exclusive-classes`, `-E` | bool | Flag to make the classification task exclusive.                                                                                                 | `False`                             |
+| `--verbose`, `-v`           | bool | Flag to print extra information to the terminal.                                                                                                | `False`                             |
+
+
+### Example usage
```

Comment:
I'd add subsections for Binary classification and Multiclass classification inside the Example usage section.